docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-linux.md

```diff
@@ -10,9 +10,194 @@ If you do not have a DirectX 11 capable GPU in your system (for example if you a
+```
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+
+5. Verify docker engine is installed by running this
+```
+sudo docker run hello-world
+```
+
+#### Debian
+1. Update `apt` package index and install packages to allow `apt` to use a repo over HTTPS
+```
+sudo apt-get update
+
+ sudo apt-get install \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+```
+
+2. Add Docker's official GPG key
+```
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+```
+
+3. Use the following commmand to set up the repository
+```
+echo \
+  " echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+```
+
+4. Update `apt` index with `sudo apt-get update` and then install docker engine, containerd, and docker compose.
+```
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+
+5. Verify docker engine is installed by running this
+```
+sudo docker run hello-world
+```
+
+#### Fedora
+Install the `dnf-plugins-core` package (which provides the commands to manage your DNF repositories) and set up the repository.
+```
+sudo dnf -y install dnf-plugins-core
+
+ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+```
+
+Install the latest version of docker engine, containerd, and docker compose
+```
+sudo dnf install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+```
+If prompted to accept the GPG key, verify that the fingerprint matches `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35` (cross reference this with the [GPG key fingerprint provided by dockers docs](https://docs.docker.com/engine/install/fedora/)) , and if so, accept it.
+
+Start docker
+```
+sudo systemctl start docker
+```
+
+Then verify that docker is installed correctly by running this
+```
+sudo docker run hello-world
+```
+
+### Prep Titanfall Server files
+
+You will need obtain Titanfall's game files, typically by copying over your Titanfall installation folder and placing it on the Linux machine. Most of these files are for SP and therefore we can delete from the installation folder and prune the file size down to \~5GB.
+
+Easiest way to do this is to copy the entire Titanfall2 folder to the server and delete the [following files](https://github.com/pg9182/northstar-dedicated#reducing-the-size-), or alternatively download and run [this sh script](https://gist.github.com/pg9182/9a962adbfc27e93237cd14e4523c9da8) made by pg9182 (note: this only downloads assets necessary to run a server and does not download a copy of the game).
+
+#### Copy Titanfall Folder over to the Linux Machine
+
+1. Access the files on your Linux machine using tools like [WinSCP](https://winscp.net/eng/download.php)
```

Comment:
Y NO filezilla anymore? ;-;

Also it reads a bit as if WinSCP is required which it really isn't anything that allows file access via SSH works. We don't want to "require" a tool but "offer" some options if one doesn't know, if you get what I mean ^^

Reply:
I personally can't recommend Filezilla since the Windows installer for it attempts to bundle in PUPs, which normally wouldn't be a huge deal but not everyone pays attention to the fine print and just lets things install, so for the sake of users I removed it.

Also yeah, it does read like that, I'll edit it to make it a bit lighter on it since it was more implied it was an option and not so much the only way with Filezilla in there.

Reply:
Updated with `Access the files on your Linux machine using tools like [WinSCP](https://winscp.net/eng/download.php) or any tool that allows for SSH file transfer, or transfer files via USB drive if you can physically access your server.` to make it clear its a suggestion, along with mentioning USB transfer for server hosts with physical access

Reply:
> Filezilla since the Windows installer for it attempts to bundle in PUPs

Huh, TIL. On Windows I usually install stuff via chocolatey and as such never got prompted cause it just pulls the "clean" version ^^